src/main/java/cz/brmlab/yodaqa/analysis/rdf/PropertyGloVeScoring.java

```diff
@@ -0,0 +1,31 @@
+package cz.brmlab.yodaqa.analysis.rdf;
+
+import cz.brmlab.yodaqa.provider.glove.MbWeights;
+import cz.brmlab.yodaqa.provider.glove.Relatedness;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by silvicek on 8/18/15.
+ */
+public class PropertyGloVeScoring {
+
+	private static PropertyGloVeScoring pgs=new PropertyGloVeScoring();
+
+	private Relatedness r=new Relatedness(new MbWeights(PropertyGloVeScoring.class.getResourceAsStream("Mbprop.txt")));
+
+	public double relatedness(String q,String prop){
```

Comment:
Could we get some brief javadoc description? What are the parameters and in what range and what meaning does the output have. (Is it between 0,1? Is higher the better?)